library/src/main/java/com/directions/route/Routing.java

```diff
@@ -27,6 +39,15 @@ protected String constructURL(LatLng... points) {
         mBuf.append(dest.longitude);
         mBuf.append("&sensor=true&mode=");
         mBuf.append(_mTravelMode.getValue());
+        if (waypoints.size() > 0) {
```

Comment:
I have just knew your waypoint pull request.
I am sorry if I offended you.
I don't plagiarize.
I'm not good at English.sorry.